### DISCUSSION TITLE
Incorrect GitHub Code Search beta search results when using `NOT is:archived`

### DISCUSSION THREAD

**Discussion by drop-nicholas** [community/community#30695] [Other Feature Feedback, Questions, & Ideas]
At `cs.github.com`, running `org:earnwithdrop NOT is:archived path:Jenkinsfile` shows files and repos that are archived.

This should not show any archived repositories, per https://cs.github.com/about/syntax#is

**Comment by DenverCoder1** [comment]
*Edit: This answer refers only to github.com search, not GitHub Code Search. The issue with GitHub Code Search still stands.*

Try using `archived:false`

See https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories#search-based-on-whether-a-repository-is-archived

**Reply by drop-nicholas** [reply]
![image](https://user-images.githubusercontent.com/100798669/186509175-30217036-f79e-4fab-8fcc-aeec5e211600.png)

No dice :game_die: :shrug:

**Reply by DenverCoder1** [reply]
It seems that I misunderstood what you were asking about. I assumed you were using github.com search and I was unaware of the GitHub Code Search beta. It seems from the link you posted that what you are doing should work, so it's possible there is a bug with that search qualifier.

**Reply by drop-nicholas** [reply]
Ah, makes sense. Yes, I think cs.github.com needs a fix :+1:

**Reply by drop-nicholas** [reply]
I updated the title. Since the cs.github.com feedback points here I didn't realize it would be so easily conflated with the generic search. Thanks for the tip on getting this to the right place there!

**Reply by DenverCoder1** [reply]
I got to try out cs.github.com with `is:archived` and `NOT is:archived` and it seems to be working the way I'd expect.

With [`user:DenverCoder1 NOT is:archived path:functions.php`](https://cs.github.com/?scopeName=All+repos&scope=&q=user%3ADenverCoder1+NOT+is%3Aarchived+path%3Afunctions.php), github-readme-youtube-stats does not show up since it is archived, but it shows up only when searching without `NOT`.

It's possible there may be an issue when searching an organization, or when the archived repos are private since those seem to be the main differences (The org in your query appears to show no archived repos on GitHub, so they must be private if there are any).

**Reply by drop-nicholas** [reply]
Must be the private org that breaks it :shrug:
